extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfoItemExtractor.java

```diff
@@ -3,17 +3,44 @@
 import org.schabi.newpipe.extractor.InfoItemExtractor;
 import org.schabi.newpipe.extractor.exceptions.ParsingException;
 import org.schabi.newpipe.extractor.localization.DateWrapper;
+import org.schabi.newpipe.extractor.stream.StreamExtractor;
 
 import javax.annotation.Nullable;
 
 public interface CommentsInfoItemExtractor extends InfoItemExtractor {
-    String getCommentId() throws ParsingException;
-    String getCommentText() throws ParsingException;
-    String getAuthorName() throws ParsingException;
-    String getAuthorThumbnail() throws ParsingException;
+
+    /**
+     * AuthorEndpoint, in other words, link to authors' channel page
+     */
```

Comment:
Imho these should be renamed to `getUploaderUrl()`, `getUploaderName()`, and `getUploaderAvatarUrl()` to be consistent with the other `Extractor`s.